cros_gralloc/cros_gralloc_helpers.cc

```diff
@@ -26,6 +26,9 @@
 /* Define to match AIDL PixelFormat::R_8. */
 #define HAL_PIXEL_FORMAT_R8 0x38
 
+/* Define to match BufferUsage::GRALLOC_USAGE_PRIVATE_1. */
+#define GRALLOC_USAGE_PRIVATE_1 (1 << 29)
```

Comment:
Could you elaborate on the sematics of this flag? And maybe a more descriptive name for this flag would be better?